src/views/__tests__/EndpointExplorer.test.tsx

```diff
@@ -557,6 +559,66 @@ describe("effects", () => {
   });
 });
 
+describe("fee_stats", () => {

```

Comment:
The tests are failing. Could you please run them locally (`yarn test:jest`) to check? Otherwise, the build check will fail. We should probably run them before committing.

![image](https://user-images.githubusercontent.com/7346473/150542651-17c9a63f-4680-42b2-83ec-ab8ed554d3cb.png)

Reply:
I must have not updated the test after I modified the setup pane; I'll fix tests and comment back once all this is resolved. sometime later today.
Thank you!

Reply:
I just got to look at this, and while fixing the tests for all_fees i am seeing some other console errors while running tests. 
![image](https://user-images.githubusercontent.com/35656692/150644301-40666e29-c305-4dae-a268-19ff84e5f873.png)
I think these might be related to 29f460f not sure how anything i touched could mess up something related to signers.

Reply:
All tests pass, but there is still an error about the signature xdr unrelated to me.  That will have to be resolved in later.